x-pack/plugins/data_visualizer/public/application/common/components/expanded_row/geo_point_content_with_map/geo_point_content_with_map.tsx

```diff
@@ -57,19 +58,16 @@ export const GeoPointContentWithMap: FC<{
     }
     updateIndexPatternSearchLayer();
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [indexPattern, combinedQuery, config, mapsPlugin]);
+  }, [indexPattern, combinedQuery, config, mapsPlugin, data.query]);
 
   if (stats?.examples === undefined) return null;
   return (
     <ExpandedRowContent dataTestSubj={'dataVisualizerIndexBasedMapContent'}>
       <DocumentStatsTable config={config} />
-
-      <EuiFlexItem style={{ maxWidth: '50%' }}>
-        <ExamplesList examples={stats.examples} />
-      </EuiFlexItem>
-      <EuiFlexItem className={'dataVisualizerMapWrapper'}>
+      <ExamplesList examples={stats.examples} />
+      <ExpandedRowPanel className={'dvPanel__wrapper dvMap__wrapper'} grow={true}>
```

Comment:
Does a style need copying over to the file data visualizer page? The map isn't rendered here when viewing a file containing geo data:

![image](https://user-images.githubusercontent.com/7405507/135298096-f290cb7e-d781-4162-b8a1-8bf3ab5524b5.png)

Reply:
Thanks for catching that. Fixed here https://github.com/elastic/kibana/pull/112870/commits/858553061dde9609eccf4d50e4ef37ab80391c18